broker/src/test/java/io/zeebe/broker/transport/backpressure/NoneRequestLimiter.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.transport.backpressure;
+
+public class NoneRequestLimiter<ContextT> implements RequestLimiter<ContextT> {
```

Comment:
I think we typically use `Noop` instead of `None` when referring to a class that performs a function, and `None` when referring to data holders...but I could be wrong I guess :sweat_smile: